README.md

```diff
@@ -40,6 +40,66 @@ Use a npm installed module, and pick out the `Adaptor` member.
 cli.js execute -l language-http.Adaptor -e exp.js -s state.json
 ```
 
+
+### Execute_noCLI
+
+Used to achieve the same as the CLI/Execute function without the CLI. This way, you do not need local files corresponding to the state or expression, you can use pure javascript with the corresponding end server language to communicate with the end server. 
+
+Usage and Parameters:
+
+```
+Execute_noCLI(expression, state, lan_path)                              [required]
+expression: target expression to execute                                     [required]
+state: incoming message from source application                              [required]
+lan_path: path to the language to be used (e.g. ../language-dhis2.Adaptor)   [required]
+```
+
+Example:
+
+The function assumes that the language-http is installed as per: ```npm install https://github.com/OpenFn/language-http.git```.
```

Comment:
Should this be `language-dhis2` rather than `lanugage-http`?

Reply:
Actually it is language-http as it is always used because of the functions it provides (inc. language-common if I am not wrong). The reason why I have it like this is that advanced expressions (using alterState, get() ...) are probably used a lot, so that it would be included as default, and the other ones that are more specific (dhis2, salesforce) could be downloaded and provided locally. I am not sure if this makes sense, but I will gladly adapt it according to your feedback.